packages/server/src/controllers/commentsController.ts

```diff
@@ -0,0 +1,127 @@
+import type { Request, Response } from 'express'
+import { CommentModel } from '../models/commentModel'
+import { COMMENT_ID_ERROR, DATA_DELETED, SERVER_ERROR, TOPIC_ID_ERROR } from '../constants'
+
+class CommentsController {
+  async createComment(req: Request, res: Response) {
+    try {
+      const topicId = Number(req.body.topicId) ?? null
+      const { content, user } = req.body
+
+      if (!topicId) {
+        return res.status(404).json({
+          message: TOPIC_ID_ERROR,
+        })
+      }
+
+      const comment = await CommentModel.create({
+        content,
+        user,
+        topicId,
+      })
+
+      return res.status(201).json(comment)
+    } catch {
+      return res.status(500).json({
+        message: SERVER_ERROR,
+      })
+    }
+  }
+
+  async getComment(req: Request, res: Response) {
+    try {
+      const commentId = Number(req.params.id) ?? null
+
+      if (!commentId) {
```

Comment:
Я наоборот писал что такой вариант немного излишний, извини за misunderstanding))
```suggestion
      const { id } = req.params

      if (!Number(id)) {
```

Reply:
Конкретно в таком примере проблема в использование [?? Nullish_coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) 

Он нужен как раз в ситуации когда нам не нужны значения `null` или `undefined` в выражение слева от него. А в твоём случае результатом `Number(req.params.id)` никогда не будет `null` или `undefined`
<img width="756" alt="Screenshot 2023-05-23 at 11 52 39" src="https://github.com/fastPaws-game/fastPaws/assets/15624993/e86aa400-5940-4620-b088-313f1173b9c7">
Соответственно выражение `?? null` бессмысленное и никогда не будет достигнуть.

Но даже если допустим что конструктор `Number` может вернуть null, то в чём смысл его избежать и дать ему снова такое ж значение?)